ios/FluentUI/Tab Bar/TabBarItemView.swift

```diff
@@ -242,10 +243,16 @@ class TabBarItemView: UIControl {
         }
     }
 
-    private func updateLayout() {
-        imageView.image = item.unselectedImage(isInPortraitMode: isInPortraitMode, labelIsHidden: titleLabel.isHidden)
-        imageView.highlightedImage = item.selectedImage(isInPortraitMode: isInPortraitMode, labelIsHidden: titleLabel.isHidden)
+    private func updateImage() {
+        // Normally we'd set imageView.image and imageView.highlightedImage separately. However, there's a known issue with
+        // UIImageView in iOS 16 where highlighted images lose their tint color in certain scenarios. While we wait for a fix,
+        // this is a straightforward workaround that gets us the same effect without triggering the bug.
+        imageView.image = imageView.isHighlighted ?
```

Comment:
should we do available iOS 16?

Reply:
I thought about leaving the old code for iOS 14/15, but at the end of the day I'd rather we have one supported solution, since there's no real performance impact or other negative side-effect.